cfgov/paying_for_college/tests/test_models.py

```diff
@@ -77,9 +79,25 @@ def create_page(model, title, slug, parent):
         )
 
     def test_loan_quiz_template(self):
+        page = self.loan_quiz_page
+        stream_block = page.content.stream_block
+
+        def get_stream_value():
+            return [{
+                'type': 'guided_quiz',
+                'id': '12345',
+                'value': {
+                    'question': '?',
+                    'answer': 'huh?'}}]
+        page.content = StreamValue(
+            stream_block,
+            get_stream_value(),
+            is_lazy=True)
         self.assertEqual(
-            self.loan_quiz_page.get_template(HttpRequest()),
+            page.get_template(HttpRequest()),
             'paying-for-college/choose-a-student-loan.html')
+        self.assertEqual(
```

Comment:
It'd be nice to have a comment here explaining what this is testing. It's non-obvious that calling `get_template` on a page instance modifies StreamField block values.

Reply:
√